docs/conf.py

```diff
@@ -227,7 +227,7 @@ def _config(key):
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = '_static/img/envoy-logo.png'
+html_logo = 'img/envoy-logo.png'
```

Comment:
i think this was broken by me when updating the docs build pipeline

im wondering a few things

- should this actually be in the `_static` folder but i added this folder incorrectly
- are there any other `_static` refs that are broken
- why didnt sphinx pick up the bad link

ill investigate further...

Reply:
I actually suspect this was the Sphinx upgrade. Basically the logo is now relative to the static folder, so the _static part was duplicated.